pkg/console/controllers/healthcheck/controller.go

```diff
@@ -161,7 +172,7 @@ func (c *HealthCheckController) CheckRouteHealth(ctx context.Context, operatorCo
 				url, _, err = routeapihelpers.IngressURI(route, route.Spec.Host)
 				if err != nil {
 					reason = "RouteNotAdmitted"
-					return fmt.Errorf("console route is not admitted")
+					return fmt.Errorf("%s route is not admitted", route.Name)
```

Comment:
Could you add the logging in here before each return? I believe doing it above will only log once after the function returns, and not during the retry loop. Doing so here will show each error we hit and the timestamp, and give us something to search for to see if the problem is still happening if this retry change fixes it and the test start passsing. (which there's a very good chance of)